src/models/stocks.ts

```diff
@@ -0,0 +1,63 @@
+import axios from "axios";
+
+import { users } from "./user";
+import type { IStock } from "@interfaces";
+import { mongoose } from "@services";
+import { STOCK_API_URL } from "@src/types/values";
+
+class Stocks extends mongoose.Repository<IStock> {
+	private static readonly stockApiClient = axios.create({
+		baseURL: STOCK_API_URL
+	});
+
+	constructor() {
+		super("Stock", {
+			server_id: {
+				type: String,
+				index: true,
+				required: true
+			},
+			user: {
+				type: String,
+				ref: users.model.modelName,
+				index: true,
+				required: true
+			},
+			symbol: {
+				type: String,
+				index: true
+			},
+			price: {
+				type: Number
+			},
+			currency: {
+				type: String
+			}
+		});
+	}
+
+	public async getPrice(symbol: string): Promise<IStock> {
+		try {
+			const { data } = await Stocks.stockApiClient.get(symbol);
+
+			const priceString: string = data
```

Comment:
remove `:string`, the type is implicit

Reply:
It's actually does not seem to be implicit in this case:

![image](https://user-images.githubusercontent.com/1410481/168925758-35d7579b-a429-4fcc-8e9b-7a21553142aa.png)